scripts/build-docker.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
what's the reasoning for this?

Reply:
I have spent much time chasing around the various docker containers used in the cosmos-sdk.

That there builds them all locally. 

No chasing.